meerkat_abacus/data_import.py

```diff
@@ -213,7 +214,13 @@ def add_rows_to_db(form, form_data, session, engine,
     logging.debug("Number of records %s", i)
     return new_rows
 
-
+def flatten_structure(row):
+    """
+    Flattens all lists in row to comma separated strings"
+    """
+    for key in row:
+        if isinstance(row[key], list):
+            row[key] = ",".join(row[key])
```

Comment:
1. I think that this will perform slightly better (faster iteration and no need to recompute key hash in every iteration):
```python
for key, value in row.items():
  if isinstance(value, list):
     row[key] = ",".join(value)
```
Also add some space after this function and the next one.

1. This may be overthinked. Do we need to worry for existing `,` in values?

Reply:
I do not think we need to worry about that for now. Will update.